jetty-websocket/javax-websocket-common/src/main/java/org/eclipse/jetty/websocket/javax/common/JavaxWebSocketFrameHandler.java

```diff
@@ -217,7 +217,9 @@ public void onError(Throwable cause)
 
         if (errorHandle == null)
         {
-            LOG.warn("Unhandled Error: Endpoint " + endpointInstance.getClass().getName() + " missing onError handler", cause);
+            LOG.warn("Unhandled Error: Endpoint " + endpointInstance.getClass().getName() + " : " + cause);
```

Comment:
Always log Throwable here.

Reply:
hmmm maybe... @lachlan-roberts we probably will need to update a lot of tests to handle errors to stop needless stack traces in the test output.